Commands/MessageCount/pagination.go

```diff
@@ -0,0 +1,45 @@
+{{/*
+	This is the pagination code for the message leaderboard
+  
+	Recommended trigger: added reactions only
+	Trigger type: reaction
+*/}}
+{{/* dont touch */}}
+{{ $action := .Reaction.Emoji.Name }} {{/* The action being ran */}}
+{{ $validEmojis := cslice "➡" "⬅" }} {{/* Valid emojis */}}
+{{ $isValid := false }} {{/* Whether this is actually a valid embed / leaderboard embed */}}
+{{ $page := 1 }} {{/* The current page */}}
+{{ with and (eq .ReactionMessage.Author.ID 204255221017214977) .ReactionMessage.Embeds }} {{/* Checks for validity */}}
+	{{ $embed := index . 0 }} {{/* The first embed */}}
+	{{ if and (eq $embed.Title (print "❯ Messages leaderboard")) $embed.Footer }} {{/* More checks */}}
+		{{ $page = reFind `\d+` $embed.Footer.Text }} {{/* We presume that this is valid, and get the page num */}}
```

Comment:
too vague imo, check out my pagination cc and see how I use [click here](link/hidden info) format in the description, I use ZWS (in the form of [\u200b](pageNum) ) to ensure that it's hidden since some may not like having page numbers in display / want to change the display then annoying errors come yeet